File: pyton_files/ex1.py
class Carro:
    def __init__(self, marca,modelo,ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir(self):
        print(
            self.marca,
            self.modelo,
            self.ano
        )


carro1 = Carro('Nissan', 'Skyline Gt-r R-34', 1973)
carro2 = Carro('Ford', 'GT40', 1964)

carro1.exibir()
carro2.exibir()

File: go_files/Ex1.go
package main

import (
  "fmt"
)
type Carro struct {

  Marca string
  Modelo string
  Ano  int

}

func (c Carro) ExibirDetalhes() {
  fmt.Printf("Marca: %s\nModelo: %s\nAno: %d\n\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
	
  carro1 := Carro{Marca: "Toyota", Modelo: "Corolla", Ano: 2020}
  carro2 := Carro{Marca: "Honda", Modelo: "Civic", Ano: 2021}
  carro3 := Carro{Marca: "Ford", Modelo: "Focus", Ano: 2019}

  carro1.ExibirDetalhes()
  carro2.ExibirDetalhes()
  carro3.ExibirDetalhes()
}